### PR TITLE
system_2617_dashboard: tile_map to map

### PR DESCRIPTION
**Pull request by ishleenk17**
- Enhancement

## What does this PR do?

This PR replaces the tile_map object with map in system's SSH Log dashboard

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## How to test this PR locally

Load the newly created dashboard and we should see map as the object 

- Closes https://github.com/elastic/integrations/issues/2617

## Screenshots
Below screenshot shows map object updated in System SSH Log Dashboard

Screenshot 2022-06-02 at 3 12 03 PM

### CODE REVIEW

**Review by muthu-mps — APPROVED** [review]
LGTM!

File: packages/system/kibana/dashboard/system-5517a150-f9ce-11e6-8115-a7c18106d86a.json [hunk 1]
```diff
@@ -131,45 +104,80 @@
...
+            "name": "2:panel_2",
             "type": "visualization"
         },
         {
             "id": "system-341ffe70-f9ce-11e6-8115-a7c18106d86a",
-            "name": "panel_2",
-            "type": "visualization"
-        },
-        {
-            "id": "system-3cec3eb0-f9d3-11e6-8a3e-2b904044ea1d",
```
Comment (flash1293) [inline packages/system/kibana/dashboard/system-5517a150-f9ce-11e6-8115-a7c18106d86a.json:160]: The old `system-3cec3eb0-f9d3-11e6-8a3e-2b904044ea1d` visualization is not part of the dashboard anymore, but it's not deleted:
Screenshot 2022-06-03 at 12 24 20

File: packages/system/kibana/map/system-4217db10-e230-11ec-baf0-970634a1784d.json [hunk 2]
```diff
@@ -0,0 +1,202 @@
+{
```
Comment (flash1293) [inline packages/system/kibana/map/system-4217db10-e230-11ec-baf0-970634a1784d.json:1]: The map is saved as a separate visualization in the library. As it's not used on another dashboard it makes sense to remove it from the library and make it part of the dashboard itself - see "Do not use “library visualizations”" in the best practices document: https://docs.google.com/document/d/1uyyFGx6xA5Kvl8c-ZdvXdvBGrHTylxU9F69TGqfzdmw/edit#

**Review by flash1293 — CHANGES_REQUESTED** [review]

File: packages/system/kibana/map/system-4217db10-e230-11ec-baf0-970634a1784d.json (hunk 2 above)
Comment (ishleenk17) [reply packages/system/kibana/map/system-4217db10-e230-11ec-baf0-970634a1784d.json]: > The map is saved as a separate visualization in the library. As it's not used on another dashboard it makes sense to remove it from the library and make it part of the dashboard itself - see "Do not use “library visualizations”" in the best practices document: https://docs.google.com/document/d/1uyyFGx6xA5Kvl8c-ZdvXdvBGrHTylxU9F69TGqfzdmw/edit#

Both the changes have been taken care of. Please re-review   the changes @flash1293

**Pull request closed without merging**